integration-tests/src/test/scala/eu/neverblink/jelly/integration_tests/rdf/io/JenaStreamSerDes.scala

```diff
@@ -161,6 +175,7 @@ object JenaStreamSerDes extends NativeSerDes[Seq[Triple], Seq[Quad]], ProtocolSe
 
   override def getBlankNodeLabel(node: Node): String = node.getBlankNodeLabel
 
+  @nowarn
```

Comment:
Please use a specifier for deprecations so that it's clear what we are silencing here. Also please add a comment explaining this very briefly.